pkg/agent/clusterclaim.go

```diff
@@ -61,6 +63,16 @@ func (c *agentController) createManagementClusterClaim(ctx context.Context) erro
 	return createOrUpdate(ctx, c.spokeClustersClient, managementClaim)
 }
 
+func (c *agentController) createHostedClusterCountClusterClaim(ctx context.Context, count int) error {
+	if count > (util.MaxHostedClusterCount - 1) {
+		c.log.Info(fmt.Sprintf("ATTENTION: the hosted cluster count has reached the maximum %s.", strconv.Itoa(util.MaxHostedClusterCount)))
```

Comment:
Maybe consider `c.log.Warning` if it has that interface.

Reply:
Just Info and Error.